core/nativeaot/README.md

```diff
@@ -1,9 +1,5 @@
 # Native AOT compilation sample apps
 
-This folder contains sample code demonstrating how to build apps with Native AOT compilation. The requirements for each sample can be found within the corresponding folder, and each sample can be ran using dotnet publish.
+This folder contains sample code demonstrating how to publish apps and libraries with native AOT compilation.
 
-Native AOT compilation is an opt-in publishing mode that compiles apps to native code similar to C/C++/Go/Swift/Rust. Not all .NET code will work with native AOT compilation.
-
-## Prerequisites
-
-Up-to-date prerequisites are in the [runtime repo](https://github.com/dotnet/runtime/blob/main/src/coreclr/nativeaot/docs/prerequisites.md).
+For more information, see https://docs.microsoft.com/en-us/dotnet/core/deploying/native-aot.
```

Comment:
This will provide better SEO on the linked article. (Also, we remove the language and culture so readers get the correct localized version:

```suggestion
For more information, see the article on [native AOT deployment](https://docs.microsoft.com/dotnet/core/deploying/native-aot).
```